README.md

```diff
@@ -381,6 +381,18 @@ Names of CSS [at-rules][at-rules] to allow that may have associated blocks
 containing style rules. At-rules like `media` and `keyframes` fall into this
 category. Names should be specified in lowercase.
 
+##### :css => :at_import_url_validator
```

Comment:
Yeah, absolutely, I'll do that rename now. I was just trying to follow the existing naming but you're absolutely right.